src/pinefarm/external/__init__.py

```diff
@@ -1,55 +1,52 @@
 """External interfaces module.
 
 All imports of external interfaces are 'hidden'
-under the function ``decide_external_tool`` to avoid
-unnecessary installations of external codes.
+under the function ``decide_external_tool`` to avoid an
+eager import and thus unnecessary installations of external codes.
 """
 
 from ..configs import configs
+from .interface import External
```

Comment:
I don't think we should have this import just for the `->`

Reply:
This function should be the first entry into external, without having to worry what actually happens in `.interface`. If it were used for something is ok, but I'd rather not have an annotation here.

Reply:
fine